.travis.yml

```diff
@@ -37,10 +37,8 @@ jobs:
         - rm -rf ./travis-build
         # add auth token to .npmrc for lerna publish
         - echo "//registry.npmjs.org/:_authToken=\${NPM_TOKEN}" >> $HOME/.npmrc 2> /dev/null
```

Comment:
Theoretically you could also add this to the release script.

Reply:
@mjhenkes I added it.